src/main/java/com/gluonhq/substrate/Constants.java

```diff
@@ -117,7 +117,7 @@
     /**
      * Supported GraalVM versions
      */
-    public static final String GRAALVM_MIN_VERSION = "21.0";
+    public static final String GRAALVM_MIN_VERSION = "22.0";
```

Comment:
shouldn't this be 22.0.0.3 since the cached_scope_id field is removed?

Reply:
Hmm, we only check major.minor.patch versions, but it could be added

Reply:
not really needed though with the check on the vendor which is part of this PR

Reply:
The check will log a warning only, so those using 22.0.0.2 or lower will need to add the method to the jni list or use runAgent